docker-compose.production.yml

```diff
@@ -318,3 +318,21 @@ services:
       - "./volumes/redis/redis.conf:/usr/local/etc/redis/redis.conf"
     command: redis-server /usr/local/etc/redis/redis.conf
     restart: always
+  
+  collab-server:
+    image: cofacts/collab-server
+    ports:
+      - "5001:1234"
+    environment:
+      - PORT=1234
+      - ELASTICSEARCH_URL=http://db:9200
+      - SECRET= 
+
+  collab-server-staging:
+    image: cofacts/collab-server
+    ports:
+      - "5001:1234"
```

Comment:
When I run this on staging, 5001 is already occupied. We may consider not exposing any port to host machine.

Reply:
ok